CMakeLists.txt

```diff
@@ -44,9 +44,24 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 # External packages
 
+# In some situations (like building python wheels), it is useful to statically link to
+# our external dependencies.  This allows us to ship self-contained compiled
+# extensions.  We check a variable here, and if set, we look for static versions of
+# our dependencies.
+#
+if(NOT TOAST_STATIC_DEPS AND ENV{TOAST_STATIC_DEPS})
+  set(TOAST_STATIC_DEPS $ENV{TOAST_STATIC_DEPS})
+endif()
+
 find_package(OpenMP)
 
-find_package(BLAS)
+if(TOAST_STATIC_DEPS)
+    set(BLA_STATIC TRUE)
+    find_package(BLAS)
```

Comment:
Is it necessary to have `find_package(BLAS)` in both branches? Why not after the `if`?

Reply:
Good point, will fix that.